tests/api/cart.api.test.js

```diff
@@ -32,12 +33,10 @@ test.describe.parallel('Carts API', () => {
       data: getCartBody(idProduct),
       headers: { 'Authorization': authorization }
     })
-
-    await expect(response).toBeOK()
-
     const responseBody = JSON.parse(await response.text())
 
-    await expect(responseBody.message).toBe('Cadastro realizado com sucesso')
+    await expect(response).toBeOK()
+    expect(responseBody.message).toEqual(POST_SUCESS)
```

Comment:
@uLucasFraga let's not hardcode any text message, we should get these from the codebase instead. I managed to get the constants from serverest repository, please follow this example and apply to other tests from the PR 👍 

Please fix the following things in your test as well:

- `expect(responseBody.message)` is not async, so it doesn't need an `await`
- Let's order the expects like shown in this test
- Use `.toEqual` to expect for any primitive, `.toBe` works but it's more suited for object comparison
- Refer to [serverest constants file](https://github.com/ServeRest/ServeRest/blob/trunk/src/utils/constants.js) to identify which ones you should import and use

Reply:
oh gosh, that's amazing!
I'll try now. 🛸